src/shogun/labels/MulticlassLabels.h

```diff
@@ -57,6 +57,14 @@ class CMulticlassLabels : public CDenseLabels
 		 */
 		CMulticlassLabels(CFile* loader);
 
+		/** constructor
```

Comment:
Capital C in constructor. Also no need to say that it is constructor, so rather remove